docs/specification/draft/basic/authorization.mdx

```diff
@@ -91,8 +94,12 @@ MCP clients **MUST** be able to parse `WWW-Authenticate` headers and respond app
 
 #### 2.3.2 Server Metadata Discovery
 
-MCP clients **MUST** follow the OAuth 2.0 Authorization Server Metadata [RFC8414](https://datatracker.ietf.org/doc/html/rfc8414)
-specification to obtain the information required to interact with the authorization server.
+MCP clients **MUST** follow either:
```

Comment:
This contradicts the requirement on clients above that says clients MUST support both.